core/build/dependencies.json

```diff
@@ -16,7 +16,9 @@
                     "flags": [
                         "-DFLATBUFFERS_BUILD_FLATLIB=FALSE",
                         "-DFLATBUFFERS_BUILD_FLATHASH=FALSE",
-                        "-DFLATBUFFERS_BUILD_TESTS=FALSE"
+                        "-DFLATBUFFERS_BUILD_TESTS=FALSE",
+						"-DCMAKE_CXX_STANDARD=17",
+						"-DCMAKE_CXX_FLAGS=-Wno-stringop-overflow"
```

Comment:
This won't work on Windows, since Visual C++ doesn't recognize this flag. Instead, you'll have to specify platform-specific flags. The `flatbuffers` section of `dependencies.json` does something like that for `osx`. You'll have to create another such section with `"platforms": ["linux-x86", "linux-x64"]`.